### PR TITLE
Add content 2

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there are some missing images. #29 

**What is the new behavior (if this is a feature change)?**
- Some more images and data has been added for today's presentation. #29

### CODE REVIEW

File: src/components/organisms/Sections/About/about.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
     background-color: #f6ede0;
     padding-left: 15%;
     padding-right: 15%;
-
+
```
Comment (kleberbaum) [inline src/components/organisms/Sections/About/about.scss]: whitespace?

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/organisms/Sections/About/about.scss (hunk 1 above)
Comment (Aichnerc) [reply src/components/organisms/Sections/About/about.scss]: Just am improved bootstrap container for desktop devices. Like in any other section, yes.

Comment (kleberbaum) [reply src/components/organisms/Sections/About/about.scss]: ...wtf

**Review by Aichnerc — COMMENTED** [review]
Looks ok

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**